src/redux/actions/index.ts

```diff
@@ -5,14 +5,24 @@ export interface LoginAction {
     payload : LoginPayload
 }
 
-interface AddWorkoutPayload {
+export interface AddWorkoutAction {
```

Comment:
We can leave this here for now, but we'll probably need to separate the actions into their own files grouped by feature later